### ISSUE TITLE
Research: What happens if the list of SortedTroves reaches max size

### ISSUE THREAD

**Issue by udosson** [kumodao/kumo-protocol#191]
https://github.com/kumodao/kumo-protocol/blob/main/packages/contracts/contracts/SortedTroves.sol#L245

**Comment by arthurka-o** [comment]
Can you please provide more information about this task? @udosson

**Comment by udosson** [comment]
See update in the description

**Comment by arthurka-o** [comment]
I don't think we need to worry about getting more than `115792089237316195423570985008687907853269984665640564039457584007913129639935` vaults 😂
